2022-11-11 12:42:39,818 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Configure stats pid to 1611911
2022-11-11 12:42:39,819 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Loading settings from /home/otastet1/.config/wandb/settings
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Loading settings from /lustre06/project/6065672/otastet1/projects/prdm9_chipseq/recode/train/wandb/settings
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'api_key': '***REDACTED***', 'mode': 'dryrun'}
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_init.py:_log_setup():476] Logging user logs to /lustre06/project/6065672/otastet1/projects/prdm9_chipseq/recode/train/wandb/offline-run-20221111_124239-1qy15jik/logs/debug.log
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_init.py:_log_setup():477] Logging internal logs to /lustre06/project/6065672/otastet1/projects/prdm9_chipseq/recode/train/wandb/offline-run-20221111_124239-1qy15jik/logs/debug-internal.log
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_init.py:init():516] calling init triggers
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-11-11 12:42:39,820 INFO    MainThread:1611911 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 3jdyuwi3
2022-11-11 12:42:39,821 INFO    MainThread:1611911 [wandb_run.py:_finish():1746] finishing run test-chip/3jdyuwi3
2022-11-11 12:42:39,821 INFO    MainThread:1611911 [jupyter.py:save_history():448] not saving jupyter history
2022-11-11 12:42:39,822 INFO    MainThread:1611911 [jupyter.py:save_ipynb():378] not saving jupyter notebook
2022-11-11 12:42:39,822 INFO    MainThread:1611911 [wandb_init.py:_jupyter_teardown():408] cleaning up jupyter logic
2022-11-11 12:42:39,822 INFO    MainThread:1611911 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-11 12:42:39,822 INFO    MainThread:1611911 [wandb_run.py:_restore():1932] restore
2022-11-11 12:42:39,822 INFO    MainThread:1611911 [wandb_run.py:_restore():1938] restore done
2022-11-11 12:42:40,969 INFO    MainThread:1611911 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-11 12:42:40,969 INFO    MainThread:1611911 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
2022-11-11 12:42:40,972 INFO    MainThread:1611911 [wandb_init.py:init():569] starting backend
2022-11-11 12:42:40,972 INFO    MainThread:1611911 [wandb_init.py:init():573] setting up manager
2022-11-11 12:42:40,980 INFO    MainThread:1611911 [backend.py:_multiprocessing_setup():102] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-11-11 12:42:40,981 INFO    MainThread:1611911 [wandb_init.py:init():580] backend started and connected
2022-11-11 12:42:40,993 INFO    MainThread:1611911 [wandb_run.py:_label_probe_notebook():1113] probe notebook
2022-11-11 12:42:40,993 INFO    MainThread:1611911 [wandb_run.py:_label_probe_notebook():1123] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-11 12:42:40,993 INFO    MainThread:1611911 [wandb_init.py:init():658] updated telemetry
2022-11-11 12:42:40,995 INFO    MainThread:1611911 [wandb_init.py:init():728] starting run threads in backend
2022-11-11 12:42:42,090 INFO    MainThread:1611911 [wandb_run.py:_console_start():1980] atexit reg
2022-11-11 12:42:42,090 INFO    MainThread:1611911 [wandb_run.py:_redirect():1838] redirect: SettingsConsole.WRAP_RAW
2022-11-11 12:42:42,090 INFO    MainThread:1611911 [wandb_run.py:_redirect():1903] Wrapping output streams.
2022-11-11 12:42:42,090 INFO    MainThread:1611911 [wandb_run.py:_redirect():1925] Redirects installed.
2022-11-11 12:42:42,091 INFO    MainThread:1611911 [wandb_init.py:init():765] run started, returning control to user process
2022-11-11 12:47:53,941 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Configure stats pid to 1611911
2022-11-11 12:47:53,942 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Loading settings from /home/otastet1/.config/wandb/settings
2022-11-11 12:47:53,942 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Loading settings from /lustre06/project/6065672/otastet1/projects/prdm9_chipseq/recode/train/wandb/settings
2022-11-11 12:47:53,942 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Loading settings from environment variables: {'_require_service': 'True', 'api_key': '***REDACTED***', 'mode': 'dryrun'}
2022-11-11 12:47:53,942 INFO    MainThread:1611911 [wandb_setup.py:_flush():68] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-11 12:47:53,942 INFO    MainThread:1611911 [wandb_init.py:_log_setup():476] Logging user logs to /lustre06/project/6065672/otastet1/projects/prdm9_chipseq/recode/train/wandb/offline-run-20221111_124753-m02j3vfg/logs/debug.log
2022-11-11 12:47:53,943 INFO    MainThread:1611911 [wandb_init.py:_log_setup():477] Logging internal logs to /lustre06/project/6065672/otastet1/projects/prdm9_chipseq/recode/train/wandb/offline-run-20221111_124753-m02j3vfg/logs/debug-internal.log
2022-11-11 12:47:53,943 INFO    MainThread:1611911 [wandb_init.py:_jupyter_setup():426] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x148242d53e80>
2022-11-11 12:47:53,943 INFO    MainThread:1611911 [wandb_init.py:init():516] calling init triggers
2022-11-11 12:47:53,943 INFO    MainThread:1611911 [wandb_init.py:init():519] wandb.init called with sweep_config: {}
config: {}
2022-11-11 12:47:53,943 INFO    MainThread:1611911 [wandb_init.py:init():535] re-initializing run, found existing run on stack: 1qy15jik
2022-11-11 12:47:53,944 INFO    MainThread:1611911 [wandb_run.py:_finish():1746] finishing run test-chip/1qy15jik
2022-11-11 12:47:53,944 INFO    MainThread:1611911 [wandb_run.py:_atexit_cleanup():1949] got exitcode: 0
2022-11-11 12:47:53,944 INFO    MainThread:1611911 [wandb_run.py:_restore():1932] restore
2022-11-11 12:47:53,945 INFO    MainThread:1611911 [wandb_run.py:_restore():1938] restore done
2022-11-11 12:47:55,072 INFO    MainThread:1611911 [wandb_run.py:_footer_history_summary_info():3377] rendering history
2022-11-11 12:47:55,072 INFO    MainThread:1611911 [wandb_run.py:_footer_history_summary_info():3409] rendering summary
